feature/metis-test/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metistest/MetisServiceStub.kt

```diff
@@ -0,0 +1,42 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metistest
+
+import de.tum.informatics.www1.artemis.native_app.core.data.NetworkResponse
+import de.tum.informatics.www1.artemis.native_app.core.websocket.WebsocketProvider
+import de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.service.network.MetisService
+import de.tum.informatics.www1.artemis.native_app.feature.metis.shared.content.MetisContext
+import de.tum.informatics.www1.artemis.native_app.feature.metis.shared.content.MetisPostDTO
+import de.tum.informatics.www1.artemis.native_app.feature.metis.shared.content.dto.StandalonePost
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+
+class MetisServiceStub : MetisService {
```

Comment:
This looks like heavy java code to me. My suggestion: 
Please provide the posts attribute as a constructor parameter. A setter is not necessary, as the invoker of the constructor will be able to pass a mutable list.